en/css/README.md

```diff
@@ -273,11 +299,11 @@ in the `blog/templates/blog/post_list.html` with this:
 ```html
 <div class="content container">
     <div class="row">
-        <div class="col-md-8">
+        <div class="col">
             {% for post in posts %}
                 <div class="post">
                     <div class="date">
-                        <p>published: {{ post.published_date }}</p>
```

Comment:
What's the reason to remove `published` here?

Reply:
Uniformity. If you do the tutorial, that feel like a anomaly. Please see for that same line in rest of the tutorial.